learning/scripting/gdscript/gdscript_advanced.rst

```diff
@@ -310,9 +310,9 @@ states and quick structs:
 
     # indexing
 
-    d.mother = "rebecca" # this doesn't work (use syntax below to add a key:value pair)
+    d.mother = "rebecca" # this works
     d["mother"] = "rebecca" # this works
-    d.name = "caroline" # if key exists, assignment does work, this is why it's like a quick struct.
+    d.name = "caroline" # this works, this is why it's like a quick struct.
```

Comment:
Why keep two examples doing the same thing? The way it was written was like "haha, this doesn't work! but this does!", but if everything works, just show the two ways to index and that's it :)

Reply:
Well, I think that it is beneficial to show that it's the same thing when changing an index as when adding one.